### ISSUE TITLE
Map fails to load after update to Mojave

### ISSUE THREAD

**Issue by Masterk78** [iam4x/pokemongo-webspoof#460]
I recently updated to Mojave and when I run Webspoof, the map fails to load and remains a white screen.  Any thoughts?  I removed it from accessibility, deleted the app, redownloaded, and returned it in the accessibility menu.  Still no luck.  

It loaded once after several attempts and was an updated, more detailed map that I believe ran from Google maps, however I was then frozen.  Any thoughts?
